anta/tests/routing/isis.py

```
@@ -358,45 +318,20 @@ class VerifyISISSegmentRoutingAdjacencySegments(AntaTest):
     class Input(AntaTest.Input):
         """Input model for the VerifyISISSegmentRoutingAdjacencySegments test."""
 
-        instances: list[IsisInstance]
-
-        class IsisInstance(BaseModel):
-            """ISIS Instance model definition."""
-
-            name: str
-            """ISIS instance name."""
-            vrf: str = "default"
-            """VRF name where ISIS instance is configured."""
-            segments: list[Segment]
-            """List of Adjacency segments configured in this instance."""
-
-            class Segment(BaseModel):
-                """Segment model definition."""
-
-                interface: Interface
-                """Interface name to check."""
-                level: Literal[1, 2] = 2
-                """ISIS level configured for interface. Default is 2."""
-                sid_origin: Literal["dynamic"] = "dynamic"
-                """Adjacency type"""
-                address: IPv4Address
-                """IP address of remote end of segment."""
+        instances: list[ISISInstance]
```

IS-IS SR is only supported in default VRF.

On EOS:
```
PE11(config-router-isis-vrf-DEV)#segment-routing mpls 
% Unavailable command (not supported in non-default vrf)
```

We should add a Pydantic validator here to check if the VRF in each `ISISInstance` model is left to `default`. If not, we can raise an error and say that as of 4.33.1F, IS-IS SR is only supported in default VRF. Refer to https://www.arista.com/en/support/toi/eos-4-17-0f/13789-isis-segment-routing in the limitations section.